README.md

```diff
@@ -14,146 +14,286 @@ https://www.yelp.com/developers/documentation/v3
 npm install yelp-fusion --save
 ```
 
-## Search
+<br/>
+
+## Table of Contents
+Business Endpoint:
+  * [Business Search](#business-search)
+  * [Phone](#phone-search)
+  * [Delivery](#transaction-search)
+  * [Business Details](#business-details)
+  * [Match](#business-match)
+  * [Reviews](#reviews)
+  * [Autocomplete](#autocomplete)
+
+Event Endpoint:
+  * [Event Lookup](#event-lookup)
+  * [Event Search](#event-search)
+  * [Featured Event](#featured-event)
+
+Category Endpoint:
+  * [All Categories](#all-categories)
+  * [Category Details](#category-details)
+
+<br/>
+
+## Business Endpoint 
+
+### Business Search
 ```javascript
 'use strict';
 
 const yelp = require('yelp-fusion');
-
+const apiKey = '<YOUR API KEY>'; 
```

Comment:
Ah yes, we weren't being clear where that variable came from. Let's leave that out and just do this in the next line:
```const client = yelp.client('YOUR_API_KEY');```

Reply:
gotcha